core/src/mindustry/core/NetServer.java

```diff
@@ -299,6 +300,15 @@ private void registerCommands(){
             }
         });
 
+        clientCommands.<Player>register("a", "<message...>", "Send a message only to admins.", (args, player) -> {
+            if(!player.admin){
+                player.sendMessage("[scarlet]You must be admin to use this command.");
+                return;
+            }
+			
+            Groups.player.each(Player::admin, a -> a.sendMessage(args[0], player, "[" + Pal.adminChat + "]<A>" + NetClient.colorizeName(player.id, player.name)));
```

Comment:
this would be identical to how teamchat does it:
```suggestion
            Groups.player.each(Player::admin, a -> a.sendMessage(args[0], player, "[#" + Pal.adminChat.toString() + "]<A>" + NetClient.colorizeName(player.id, player.name)));
```